DS13/code/modules/mentor/follow.dm

```diff
@@ -0,0 +1,26 @@
+/client/proc/mentor_follow(mob/living/M)
+	if(!is_mentor())
+		return
+	if(isnull(M))
+		return
+	if(!ismob(usr))
+		return
+	mentor_datum.following = M
+	usr.reset_perspective(M)
+	verbs += /client/proc/mentor_unfollow
+	to_chat(GLOB.admins, "<span class='mentor'><span class='prefix'>MENTOR:</span> <EM>[key_name(usr)]</EM> is now following <EM>[key_name(M)]</span>")
```

Comment:
Dont think youve added this mentor span to the stylesheets for normal + dark mode chat themes

Reply:
Added